src/test/java/redis/clients/jedis/commands/jedis/ClusterCommandsTest.java

```diff
@@ -189,6 +192,34 @@ public void clusterSlots() {
     node1.clusterDelSlots(3000, 3001, 3002);
   }
 
+  @Test
+  public void clusterShards() {
+    assertEquals("OK", node1.clusterAddSlots(3100, 3101, 3102, 3105));
+
+    List<ClusterShardInfo> shards = node1.clusterShards();
+    assertNotNull(shards);
+    assertTrue(shards.size() > 0);
+
+    for (ClusterShardInfo shardInfo : shards) {
+      assertNotNull(shardInfo);
+
+      assertTrue(shardInfo.getSlots().size() > 1);
```

Comment:
````
Error:  redis.clients.jedis.commands.jedis.ClusterCommandsTest.clusterShards -- Time elapsed: 0.013 s <<< FAILURE!
java.lang.AssertionError
	...
	at org.junit.Assert.assertTrue(Assert.java:53)
	at redis.clients.jedis.commands.jedis.ClusterCommandsTest.clusterShards(ClusterCommandsTest.java:206)
	...

Reply:
You can try cleaning. But removing this assertion seems fine too.

Reply:
I added cleaning. Locally the problem has gone away. Let's try to run checks.